src/main/java/minijava/ir/optimize/AliasAnalyzer.java

```diff
@@ -824,6 +837,7 @@ public boolean transform() {
       Node ptr = sideEffect.getPred(1);
       Set<IndirectAccess> aliasClass = getPointsTo(ptr);
       boolean isLoad = sideEffect.getOpCode() == iro_Load;
+      assert !aliasClass.isEmpty() : "Empty alias class at " + sideEffect;
```

Comment:
I *think* they are enabled in the `run` script. At least it works when going through `mjtest`.